packages/widgets/src/blocks/legacy-widget/edit/preview.js

```diff
@@ -11,9 +11,19 @@ import { addQueryArgs } from '@wordpress/url';
 import { useState } from '@wordpress/element';
 import { Placeholder, Spinner, Disabled } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import { store as blockEditorStore } from '@wordpress/block-editor';
 
 export default function Preview( { idBase, instance, isVisible } ) {
 	const [ isLoaded, setIsLoaded ] = useState( false );
+	const settings = useSelect( ( select ) => {
```

Comment:
Nitpick: we don't need all the editor settings here, so maybe we can just destruct the admin url.

```suggestion
	const { wpAbsoluteAdminUrl } = useSelect( ( select ) => {
```

Reply:
Yes, I agree.
Fixed in 36370d9820206909f8821e746d56c2f3af25f8c2